lib/fprotect/Kconfig

```diff
@@ -5,11 +5,12 @@
 #
 
 config FPROTECT
-	bool "Write-protect flash regions"
+	bool "FPROTECT protects flash regions from writes and also optionally from reads."
```

Comment:
The new phrasing does not seem like a prompt. It does not describe an option in its new form, it just states information about FPROTECT.

Could we state `Protect flash regions` instead, and then have the rest of the details in the helptext?

Reply:
Hmm, no flash regions are not necessarily protected when FPROTECT is enabled.

It's only when FPROTECT and also MCUBOOT is enabled that some flash gets protected.

Perhaps I could have the prompt say, "Build the fprotect library which may be used to protect flash regions from writes and optionally from reads."

Or perhaps you have an even better idea?

Reply:
Or,

"Build the fprotect library"

in the prompt.

And details in the help text ?

Reply:
"Enable FPROTECT" (I have never seen the use of "Build ..." in these types of prompts), and then details in the help text would work.

Reply:
It know says "Enable FPROTECT".

And the help text notably and intentionally does not say that flash will be protected, as that depends on something actually using the library to protect flash.